### PR TITLE
Updated README.md and license in all files

### PR DESCRIPTION
**Pull request by maxipavlovic**
+ Also raised coverage to 100%

### CODE REVIEW

**Review by marcserrat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -24,6 +25,24 @@
...
+
+On MacOs:
+* Install system dependencies
+```commandline
+brew install py3cairo pango
+```
+* Create virtualenv
+* Install project dependencies
+```commandline
+pip insatll poetry
```
Comment (ffaraoneim) [inline README.md:38]: Please fix mispelled install

File: README.md [hunk 2]
```diff
@@ -24,6 +25,24 @@
...
+poetry update
+```
+* Run tests
+```commandline
+pytest
```
Comment (ffaraoneim) [inline README.md:43]: If poetry virtualenv is enabled this should be: 

```
poetry run pytest
```

**Review by ffaraoneim — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maxipavlovic) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (maxipavlovic) [reply README.md]: Fixed

**Review by ffaraoneim — APPROVED** [review]

**Pull request merged**